Compile error on Windows 10 with Godot 3.1

I am unable to compile Godot engine with voxel module as I am getting following error:

```
[Initial build] Compiling ==> modules\voxel\streams\voxel_block_serializer.cpp
voxel_block_serializer.cpp
modules\voxel\streams\voxel_block_serializer.cpp(90): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(119): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(164): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(167): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
scons: *** [modules\voxel\streams\voxel_block_serializer.windows.opt.tools.64.obj] Error 2
scons: building terminated because of errors.
```

I am compiling on Windows 10 Pro 64bit. I tried to compile using MinGW and MSVC with same result:
```
scons platform=windows target=release_debug
scons platform=windows target=release_debug use_mingw=yes
```

I have no problem to compile Godot without voxel module.

Compiling godot/master branch works without a problem. But since I have no idea how stable it is, I'd rather stick to the godot-3.1.1-stable release.

Adding `#define ERR_FAIL_COND_V_MSG` and `#define ERR_FAIL_COND_MSG` to 2 files did solve this issue, but then I hit another error:

```
[Initial build] Compiling ==> modules\voxel\util\direct_static_body.cpp
direct_static_body.cpp
modules\voxel\util\direct_static_body.cpp(48): error C2660: 'PhysicsServer::body_add_shape': function does not take 4 arguments
E:\Workspace\Godot\godot-3.1.1-stable\servers/physics_server.h(375): note: see declaration of 'PhysicsServer::body_add_shape'
scons: *** [modules\voxel\util\direct_static_body.windows.tools.64.obj] Error 2
```

And after fixing this issue, there is a great chance, there will be another one. So as far as I know, godot_voxel/master works only with Godot/master, not with latest stable release :(

> Compiling godot/master branch works without a problem. But since I have no idea how stable it is, I'd rather stick to the godot-3.1.1-stable release.

I regularly and severely abuse it and don't have any stability issues.

> modules\voxel\util\direct_static_body.cpp(48): error C2660: 'PhysicsServer::body_add_shape': function does not take 4 arguments

Remove the `, false` from that line. I think that is to add the shape in a disabled state, which is unnecessary.

> And after fixing this issue, there is a great chance, there will be another one. 

Maybe, but let's not worry about something that might never happen.

> So as far as I know, godot_voxel/master works only with Godot/master, not with latest stable release :(

Godot_voxel is itself alpha. However is very stable already. And new, incredible features have been added recently, so while you could go to an older revision that will compile natively against 3.1, you'll miss out on a lot. So again, let's not worry over nothing. When there's something tangible, like a compile error, then you can worry about that.

--- a/error_macros.h
+++ b/error_macros.h
@@ -1,0 +1,22 @@
+// Macros defined in Godot 3.2 in core/error_macros.h
+// Copied here for Voxel Tools 3.1 branch
+
+#define ERR_FAIL_COND_MSG(m_cond, m_msg)                                                                   \
+	{                                                                                                      \
+		if (unlikely(m_cond)) {                                                                            \
+			ERR_EXPLAIN(m_msg);                                                                            \
+			_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "Condition ' " _STR(m_cond) " ' is true."); \
+			return;                                                                                        \
+		}                                                                                                  \
+		_err_error_exists = false;                                                                         \
+	}
+
+#define ERR_FAIL_COND_V_MSG(m_cond, m_retval, m_msg)                                                                                 \
+	{                                                                                                                                \
+		if (unlikely(m_cond)) {                                                                                                      \
+			ERR_EXPLAIN(m_msg);                                                                                                      \
+			_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "Condition ' " _STR(m_cond) " ' is true. returned: " _STR(m_retval)); \
+			return m_retval;                                                                                                         \
+		}                                                                                                                            \
+		_err_error_exists = false;                                                                                                   \
+	}

--- a/streams/voxel_block_serializer.cpp
+++ b/streams/voxel_block_serializer.cpp
@@ -1,4 +1,5 @@
 #include "voxel_block_serializer.h"
+#include "../error_macros.h"
 #include "../math/vector3i.h"
 #include "../thirdparty/lz4/lz4.h"
 #include "../voxel_buffer.h"

--- a/streams/voxel_stream_region_files.cpp
+++ b/streams/voxel_stream_region_files.cpp
@@ -1,4 +1,5 @@
 #include "voxel_stream_region_files.h"
+#include "../error_macros.h"
 #include "../math/rect3i.h"
 #include "../util/utility.h"
 #include "file_utils.h"

--- a/terrain/voxel_map.cpp
+++ b/terrain/voxel_map.cpp
@@ -1,6 +1,7 @@
 #include "voxel_map.h"
 #include "../cube_tables.h"
 #include "voxel_block.h"
+#include "../error_macros.h"
 
 #include "core/os/os.h"
 

--- a/util/direct_static_body.cpp
+++ b/util/direct_static_body.cpp
@@ -45,7 +45,7 @@ void DirectStaticBody::set_transform(Transform transform) {
 
 void DirectStaticBody::add_shape(Ref<Shape> shape) {
 	ERR_FAIL_COND(!_body.is_valid());
-	PhysicsServer::get_singleton()->body_add_shape(_body, shape->get_rid(), Transform(), false);
+	PhysicsServer::get_singleton()->body_add_shape(_body, shape->get_rid(), Transform());
 	// No use case for multishape yet
 	_shape = shape;
 

